.eslintrc.json

```diff
@@ -0,0 +1,71 @@
+{
+	// Provide predefined global variables
+	"env": {
+		"browser": true, // browser global variables
+		"es2021": true, // all ECMAScript 2021 globals
+		"node": true // Node.js global variables and Node.js scoping
+	},
+	"extends": "eslint:recommended",
+	"parserOptions": {
+		"ecmaVersion": "latest", // The version of ECMAScript to support — the most recent supported version
+		"sourceType": "module" // The type of JavaScript source code — ECMAScript modules (ESM)
+	},
+	"rules": {
+		// Tabbed indentation (https://eslint.org/docs/latest/rules/indent)
+		"indent": [
+			"error",
```

Comment:
I might be in the minority, but personally I don't think any linting issues should be reported as errors, as this detracts from things that are actual, breaking, errors. I tend to define all my ESLint rules to be warnings.

Reply:
I'm not against going with warnings then. :)